components/infobox/wikis/tft/infobox_league_custom.lua

```diff
@@ -63,6 +64,10 @@ end
 
 function CustomInjector:parse(id, widgets)
 	if id == 'gamesettings' then
+		table.insert(widgets, Cell{
+			name = 'Game Version',
```

Comment:
```suggestion
			name = 'Game',
```
Wouldn't think make more sense?
Game Version sounds pretty weird for a multi-game wiki no?

Reply:
I actually agree with this, i've updated the new revision.